ecosystem/indexer-grpc/indexer-grpc-data-service/src/test_lib/mod.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_protos::transaction::v1::{
+    transaction::TxnData, transaction_payload::Payload, EntryFunctionId, EntryFunctionPayload,
+    Event, MoveModuleId, Signature, Transaction, TransactionInfo, TransactionPayload,
+    UserTransaction, UserTransactionRequest, WriteSetChange,
+};
+
+#[allow(dead_code)]
```

Comment:
Can just define this within the `#[cfg(test)]` block in `service.rs` along with the other unit tests so it doesn't have to be in a separate file and u won't have to use this tag 😄